activities/Moon.activity/js/moon-activity.js

```diff
@@ -1,7 +1,7 @@
 define(['activity/data-model', 'activity/draw', 'webL10n', 'sugar-web/env', 'sugar-web/datastore', 'moment-with-locales.min'], function(DataModel, Draw, l10n, env, datastore, moment) {
 
     'use strict';
-
+    requirejs(['humane'], function (doc, humane) { 
```

Comment:
That's not correct, `humane` should be added in the `define` method above.

Reply:
sorry for that, did the change no difference